packages/sdk/contractkit/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@celo/contractkit",
-  "version": "1.3.3-dev",
+  "version": "1.3.4-dev",
```

Comment:
It looks like this might be breaking the build because other monorepo packages rely on 1.3.3-dev